stories/Dropdown.js

```diff
@@ -0,0 +1,189 @@
+      console.log('click');
+    }
+
+    return (
+      <Dropdown isActive>
+        <DropdownItem title="Dropdown item 1" onClick={onItemClick} active />
+        <DropdownItem title="Dropdown item 2" onClick={onItemClick} />
+        <DropdownItem title="Dropdown item 3" onClick={onItemClick} />
+        <DropdownItem
+          title="Dropdown item 4. Link, internal"
+          href="/"
+        />
+        <DropdownItem
+          title="Dropdown item 5. Link, external"
+          href="https://droplr.com"
+          target="_blank"
+        />
+      </Dropdown>
+    );
+  })
+  .addWithJSX('Dropdown no item active state', () => {
+    function onItemClick(e) {
+      console.log('click');
+    }
+
+    return (
+      <Dropdown isActive noItemsActiveState>
+        <DropdownItem title="Dropdown item 1" onClick={onItemClick} active />
+        <DropdownItem title="Dropdown item 2" onClick={onItemClick} />
+        <DropdownItem title="Dropdown item 3" onClick={onItemClick} />
+        <DropdownItem
+          title="Dropdown item 4. Link, internal"
+          href="/"
+        />
+        <DropdownItem
+          title="Dropdown item 5. Link, external"
+          href="https://droplr.com"
+          target="_blank"
+        />
+      </Dropdown>
+    );
+  })
+  .addWithJSX('Dropdown with title', () => {
+    function onItemClick(e) {
+      console.log('click');
+    }
+
+    return (
+      <Dropdown header="Dropdown title" isActive >
+        <DropdownItem title="Dropdown item 1" onClick={onItemClick} active />
+        <DropdownItem title="Dropdown item 2" onClick={onItemClick} />
+        <DropdownItem title="Dropdown item 3" onClick={onItemClick} />
+        <DropdownItem
+          title="Dropdown item 4. Link, internal"
+          href="/"
+        />
+        <DropdownItem
+          title="Dropdown item 5. Link, external"
+          href="https://droplr.com"
+          target="_blank"
+        />
+      </Dropdown>
+    );
+  })
+  .addWithJSX('Complex', () => {
+    function onItemClick(e) {
+      console.log('click');
+    }
+
+    return (
+      <Dropdown isActive >
+        <DropdownItem
+          title={{ text: 'Item 1', Icon: PadlockLockIcon }}
+          onClick={onItemClick}
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor"
+          active
+          disabled
+        />
+        <DropdownItem
+          title={{ text: 'Item 2', Icon: TeamIcon }}
+          onClick={onItemClick}
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor"
+          disabled
+        />
+        <DropdownItem
+          title={{ text: 'Item 3', Icon: PadlockUnlockIcon }}
+          onClick={onItemClick}
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor"
+        />
+        <DropdownItem className="drui-dropdownItem--danger" title="Delete" onClick={onItemClick} Icon={TrashBinIcon} />
```

Comment:
Why have you added a custom class to that component and added some custom styling to it?
Those stories should be a show of what can we use from here.
Such custom code in the stories seems to me like a bad practice.